prow/config.yaml

```diff
@@ -43,6 +43,19 @@ deck:
   hidden_repos:
   - kubernetes-security
   google_analytics: UA-82843984-5
+  rerun_auth_config:
```

Comment:
I forget where we do script yaml checking. Any chance this blows things up? AKA should we wait until we push the code to read this field before deploying the field?

Reply:
I tried it on a test instance and it was fine. I can wait to do this in the next PR though if you'd prefer?

Reply:
We deliberately don't do strict yaml checking on loading config files to prevent this sort of thing causing problems (more specifically to prevent rollbacks causing problems, but this is also fine).

Reply:
Don't you run the `checkconfig` unknown fields check?